README.md

```diff
@@ -1 +1,148 @@
-# animePage
\ No newline at end of file
+<a name="readme-top"></a>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Futures](#key-futures)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Clone the repository](#Clone-the-repository)
+  - [Setup](#setup)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 Capstone Proyect (OtakuZone) <a name="about-project"></a>
+
+**In this final project I show what I learned during the first module of the program.** 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- **HTML**
+- **CSS**
+- **JavaScript**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Future <a name="key-future"></a>
+
+- **The pertinent corrections requested by the reviewer will be added**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [See Live Demo](https://andresespi.github.io/animePage/)
+
```

Comment:
- [ ] Kindly make sure that the project is deployed properly, all pages, links, and images work as expected. But in the case of your project the links and images are not working as expected as shown with the below screenshot . Kindly Fix it.
![github](https://github.com/AndresEspi/animePage/assets/93598189/3173f46a-36c2-489b-b45a-79c0812281ca)

Reply:
hello soon it is a problem with your browser since it downloaded on another pc and if you see the stylized css I recommend that you review it again